test/unit_test/passes/onnx/test_conversion.py

```diff
@@ -1,3 +1,8 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
```

Comment:
In other places, we don't have an empty line after the license header. Should we remove them here too?
Same for the other pass tests in this PR.

Reply:
nvm, there are multiple places in olive already with new lines after header. We can standardize later if needed.